pydocstringformatter/run.py

```diff
@@ -9,13 +9,17 @@
 from typing import List, Union
 
 from pydocstringformatter import __version__, formatting, utils
+from pydocstringformatter.formatting import FORMATTERS
 
 
 class _Run:
     """Main class that represent a run of the program"""
 
     def __init__(self, argv: Union[List[str], None]) -> None:
         self.arg_parser = utils._register_arguments(__version__)
+        self.arg_parser = utils._register_arguments_formatters(
```

Comment:
I believe this doesn't actually need to be re-assigned, as the object is changed in place by `add_argument`. Just a very small optimisation 😄